test/extended/util/test.go

```diff
@@ -67,7 +68,6 @@ func InitTest(dryRun bool) error {
 	testfiles.AddFileSource(conformancetestdata.GetConformanceTestdataFS())
 	TestContext.KubectlPath = "kubectl"
 	TestContext.KubeConfig = KubeConfigPath()
-	os.Setenv("KUBECONFIG", TestContext.KubeConfig)
```

Comment:
KubeConfigPath()'s only content is os.Getenv("KUBECONFIG") so this set is meaningless.